ansible/roles/gelato-ha-installer/scenarios/release.yml

```diff
@@ -32,3 +32,57 @@
   when:
     - gelatoexisted.stat.exists is undefined or gelatoexisted.stat.exists == false
 
+- name: Update the release version in deployment ymls
```

Comment:
Please consider moving configurations out of ansible installation. For example, user may not want to use ansible for the deployment. Please check with Amit/Asif for inputs

Reply:
Right now, for this release, Multi-Cloud is installable using Ansible only. Will enhance it to use other methods in the coming release. Will open an issue to track this.

Reply:
Opened #443 to track this.

Reply:
opened #443 for tracking this.